src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+export default function Display(props) {
+  const { result } = props;
+  return (
+    <p>{ result }</p>
+  );
+}
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
```

Comment:
- [ ] Should we wrap the result in quotes as a string since you have assigned the result to be string?

Reply:
Thank you for the response @mosaaleb.
The issue has been fixed.